src/txn/txn_recover.c

```diff
@@ -959,6 +975,14 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
 done:
     /* Close cached cursors, rollback-to-stable asserts exclusive access. */
     WT_ERR(__recovery_close_cursors(&r));
+#ifndef WT_STANDALONE_BUILD
+    if (conn->unclean_shutdown && conn->recovery_major == 10 && conn->recovery_minor == 0 &&
```

Comment:
Do we care about `conn->recovery_major < 10` here?

Reply:
All the problematic MongoDB versions are with version 10.0.0, I don't think we need to handle the older versions.